apps/cookbook/src/app/showcase/colors-showcase/colors-showcase.component.html

```diff
@@ -80,3 +85,20 @@ <h3>{{ color.name | slice: 0:2 | uppercase }}</h3>
     </li>
   </ul>
 </ng-template>
+
+<ng-template #decorationColorPalette let-colors>
+  <div class="color-palette-ramps">
+    <div *ngFor="let color of colors" class="color-box">
+      <h2 class="color-name">{{ color.name }}</h2>
```

Comment:
```suggestion
      <h3 class="color-name">{{ color.name }}</h3>
```
Probably irrelevant given the current state of our docs but I think this should be a h3